int_experian/int_experian_sgjc/cartridge/static/default/js/edqUtils.js

```diff
@@ -298,40 +305,38 @@ for (i = 0; i < selectSelector.length; i++) {
 }
 for (i = 0; i < buttonSelector.length; i++) {
 	if (buttonSelector[i].name.match(/dwfrm_profile_address_create/)) {
-		edqCurrentSubmitButtonSelector = buttonSelector[i];
+		edqCurrentSubmitButton = buttonSelector[i];
 	}
 	if (buttonSelector[i].name.match(/dwfrm_profile_address_edit/)) {
-		edqCurrentSubmitButtonSelector = buttonSelector[i];
+		edqCurrentSubmitButton = buttonSelector[i];
 	}
 	if (buttonSelector[i].name.match(/dwfrm_singleshipping_shippingAddress_save/)) {
-		edqCurrentSubmitButtonSelector = buttonSelector[i];
+		edqCurrentSubmitButton = buttonSelector[i];
 	}
 	if (buttonSelector[i].name.match(/dwfrm_profile_confirm/)) {
-		edqCurrentSubmitButtonSelector = buttonSelector[i];
+		edqCurrentSubmitButton = buttonSelector[i];
```

Comment:
With these matches, is there ever a situation where the button would be switched twice? For example, on the same page would `dwfrm_profile_address_create` and `dwfrm_singleshipping_shippingAddress_save` both be matched?

Reply:
For SGJC no; but for SFRA there's a case on the checkout page that the button changes by JS functions and the page doesn't reloads; this change is managed by event listeners that makes the stage change for all variables.